src/odemis/driver/scanner.py

```diff
@@ -86,17 +90,28 @@ def __init__(self, name, role, dependencies, **kwargs):
         # VAs that could be both on internal or external. If on both, pick internal
         # TODO: add a better way to select if both provide: either via arg, or
         # select the one which provides a None (=auto), or which is not read-only?
-        for vaname in ("power", "blanker", "external", "rotation"):
+        for vaname in ("power", "external", "rotation"):
             if model.hasVA(self._internal, vaname):
                 va = getattr(self._internal, vaname)
                 setattr(self, vaname, va)
             elif model.hasVA(self._external, vaname):
                 va = getattr(self._external, vaname)
                 setattr(self, vaname, va)
 
+        self.blanker = model.VAEnumerated(
+            None,
+            setter=self._setBlanker,
+            choices={True: 'blanked', False: 'unblanked', None: 'auto'})
+
```

Comment:
You should do the blanker only if the "internal" has a blanker. Possibly, you could even check if "external" has already a blanker with None, in which case all this is not needed, and .blanker should be just a copy of external.blanker.

Reply:
I think I fixed this but I'm not sure @pieleric can you check?